### ISSUE TITLE
Error: ER_BAD_FIELD_ERROR: Unknown column 'datetime_precision' in 'field list'

### ISSUE THREAD

**Issue by steverobbins** [TehShrike/joi-sql#1]
When executing I get the following error

```
$ joi-sql --host=127.0.0.1 --user=root --password=root --schema=test --table=test
/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/Parser.js:82
        throw err;
              ^
Error: ER_BAD_FIELD_ERROR: Unknown column 'datetime_precision' in 'field list'
    at Query.Sequence._packetToError (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/sequences/Sequence.js:48:14)
    at Query.ErrorPacket (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/sequences/Query.js:83:18)
    at Protocol._parsePacket (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/Protocol.js:271:23)
    at Parser.write (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/Parser.js:77:12)
    at Protocol.write (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/Protocol.js:39:16)
    at Socket.<anonymous> (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/Connection.js:92:28)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
    at TCP.onread (net.js:538:20)
    --------------------
    at Protocol._enqueue (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/protocol/Protocol.js:135:48)
    at Connection.query (/usr/local/lib/node_modules/joi-sql/node_modules/mysql/lib/Connection.js:197:25)
    at module.exports (/usr/local/lib/node_modules/joi-sql/fetch.js:15:5)
    at Object.<anonymous> (/usr/local/lib/node_modules/joi-sql/index.js:18:2)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
```

Using MySQL Version 5.1.73

**Comment by TehShrike** [comment]
Fixed by a16f546f84f25377349de51926ad515569c12701

**Comment by TehShrike** [comment]
Thanks for the issue report :-)
